compose.yml

```diff
@@ -0,0 +1,46 @@
+services:
+
+  bootstrap:
+    build:
+      context: .
+      dockerfile: Dockerfile.libp2p
+    ports:
+      - "3000:3000"
+      - "8080:8080"
+    volumes:
+      - ./testnet:/etc/nomos
+    environment:
+      - NET_NODE_KEY=1000000000000000000000000000000000000000000000000000000000000000
+    entrypoint: sh
```

Comment:
The script already has a shebang. And really, the script is the entrypoint.

Reply:
The entrypoint in `Dockerfile.libp2p` is set straight to run the node itself. The explicit `sh` is leftover when the `run_libp2p.sh` didn't have the shebang, will fix.